src/pages/Address/AddressComp.tsx

```diff
@@ -260,12 +278,38 @@ export const AddressTransactions = ({
 
   return (
     <>
-      <TitleCard title={`${i18n.t('transaction.transactions')} (${localeNumberString(total)})`} isSingle />
+      <TitleCard
+        title={`${i18n.t('transaction.transactions')} (${localeNumberString(total)})`}
+        className={styles.transactionTitleCard}
+        isSingle
+        rear={
+          <Radio.Group
+            className={styles.layoutButtons}
+            options={[
+              { label: i18n.t('transaction.professional'), value: 'professional' },
+              { label: i18n.t('transaction.lite'), value: 'lite' },
+            ]}
+            onChange={({ target: { value } }) => onChangeLayout(value)}
+            value={layout}
+            optionType="button"
+            buttonStyle="solid"
+          />
+        }
+      />
       <AddressTransactionsPanel>
+        {layout === 'lite' && !isMobile && (
+          <div className={styles.liteTransactionHeader}>
+            <div>{i18n.t('transaction.transaction_hash')}</div>
+            <div>{i18n.t('transaction.height')}</div>
+            <div>{i18n.t('transaction.time')}</div>
+            <div>{`${i18n.t('transaction.input')} & ${i18n.t('transaction.output')}`}</div>
+            <div>{i18n.t('transaction.capacity_change')}</div>
+          </div>
+        )}
         {txList.map((transaction: State.Transaction, index: number) => {
           const { transactionHash } = transaction
-          return (
-            transaction && (
+          if (transaction && layout === 'professional') {
```

Comment:
`transaction` can't be empty here, it should be legacy code.
Also can be changed to ternary syntax to improve readability.